config/iao.yml

```diff
@@ -68,8 +73,18 @@ entries:
 - prefix: /dev/
   replacement: https://raw.githubusercontent.com/information-artifact-ontology/IAO/master/src/ontology/
 
+- exact: /dev/iao-edit.owl
+  replacement: https://raw.githubusercontent.com/information-artifact-ontology/IAO/master/src/ontology/iao-edit.owl 
```

Comment:
The previous `prefix` entry covers this, so this `exact` entry will never be used.